src/huggingface_hub/hf_api.py

```diff
@@ -410,6 +410,8 @@ def erase_from_credential_store(username=None):
 
 class HfApi:
     def __init__(self, endpoint=None):
+        if endpoint is None:
+            endpoint = os.getenv('HF_ENDPOINT')
         self.endpoint = endpoint if endpoint is not None else ENDPOINT
```

Comment:
maybe more Pythonic to do those three lines in one line
```python
self.endpoint = endpoint if endpoint is not None else os.getenv('HF_ENDPOINT', ENDPOINT)
```

Reply:
other than that LGTM=)

Reply:
ok!